pkg/daemon/config.go

```diff
@@ -92,3 +103,38 @@ func tryToLoadOldConfig(nodeProfilesJson []byte) ([]ptpv1.PtpProfile, bool) {
 
 	return []ptpv1.PtpProfile{*ptpConfig}, true
 }
+
+func populatePtp4lConf(config *string, output *ptp4lconf) {
```

Comment:
same as before, I think you don't need to use *string, string should be enough.
and about changing the functions to methods it could be a bit more clear than the current implementation,
i.e func (p *ptp4lconf) Populate(config string) { ...}

Reply:
updated

Reply:
Hi I don't see the update here its still a function that is not part of a struct like this `func (p *ptp4lconf) Populate(config string) { ...}`